src/main/java/co/cask/hydrator/salesforce/plugin/source/batch/SalesforceBatchSource.java

```diff
@@ -74,11 +78,12 @@
     @Macro
     private final String query;
 
+
```

Comment:
extra newline

Reply:
Not sure why diff showed it like that. But I am looking at code and at link at https://github.com/data-integrations/salesforce/pull/4/files
and it shows only one line there.